openvpn/cyberghost/default.ovpn

```diff
@@ -0,0 +1,31 @@
+client
+remote 87-1-fr.cg-dialup.net 443
+dev tun 
+proto udp
+auth-user-pass
+
+
+resolv-retry infinite 
+redirect-gateway def1
+persist-key
+persist-tun
+nobind
+cipher AES-256-CBC
+ncp-disable
+auth SHA256
+ping 5
+ping-exit 60
+ping-timer-rem
+explicit-exit-notify 2
+script-security 2
+remote-cert-tls server
+route-delay 5
+verb 4
+
+
+ca /etc/openvpn/custom/ca.crt
```

Comment:
the crt and key files should be put under a cyberghost folder,
create a new folder in openvpn git folder and then set the path here to:
/etc/openvpn/cyberghost/ca.crt etc..

Reply:
Yes I'll do that

Reply:
Are you planning to update the PR with the fixes so it can be merged??

Reply:
Yes but not before 3 weeks because I'm not at home at the moment...